apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-international.component.ts

```diff
@@ -0,0 +1,221 @@
+/**
```

Comment:
If we are pulling in moment to get these values, i'm not sure why we have this file with the same / similar information?

Reply:
For translating Calendar, we have two options.
1. momentJs
2. Services by Calendar Api, which expects json with month, days, full months name.
So this file is an example for approach 2.